available-connectors.json

```diff
@@ -191,6 +201,42 @@
     "supportedByStrongLoop": true
   },
   {
+    "name": "kv-redis",
+    "description": "Redis key-value connector",
+    "baseModel": "KeyValueModel",
+    "features": {
+      "discovery": false,
+      "migration": false
+    },
+    "settings": {
+      "url": {
+        "type": "string",
+        "description": "Connection String url to override other settings (eg: redis://user:pass@host:port/db)"
+      },
+      "host": {
+        "type": "string"
+      },
+      "port": {
+        "type": "number"
+      },
+      "user": {
+        "type": "string"
+      },
+      "pass": {
```

Comment:
Should be `password`, because that's the value expected by https://github.com/luin/ioredis/blob/master/API.md#new_Redis_new AFAICT.